Streetcode/Streetcode.XIntegrationTest/ControllerTests/AdditionalContent/SubtitleControllerTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.AspNetCore.Mvc.Testing;
+using Streetcode.BLL.DTO.AdditionalContent.Coordinates.Types;
+using Streetcode.BLL.DTO.AdditionalContent.Subtitles;
+using Streetcode.XIntegrationTest.ControllerTests.Utils;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http.Json;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Streetcode.XIntegrationTest.ControllerTests.AdditionalContent
+{
+    public class SubtitleControllerTests: BaseControllerTests, IClassFixture<CustomWebApplicationFactory<Program>>
+    {
+        public SubtitleControllerTests(CustomWebApplicationFactory<Program> factory) : base(factory)
+        {
+
+        }
+
+        [Fact]
+        public async Task SubtitleControllerTests_GetAllSuccessfulResult()
+        {
+            var responce = await _client.GetAsync($"/api/Subtitle/GetAll");
```

Comment:
Let's not use default http client here, please create your own RestClient with all methods and routs refined there, so it will return some rest response and we can verify it here
You can use RestSharp